src/lib/sql/sql_translator.cpp

```diff
@@ -194,6 +194,8 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_statement(const hsql:
     case hsql::kStmtExport:
       return _translate_export(static_cast<const hsql::ExportStatement&>(statement));
     case hsql::kStmtTransaction:
+      // The transaction statements are handled directly in the SQLPipelineStatement,
+      //  but the translation is still called, so we need some kind of handling here.
```

Comment:
... so we return a dummy node here?

Reply:
Is this a proposal to improve the comment?

Reply:
Done.